rfc-parser/docs/ModelDeviations.md

```diff
@@ -43,36 +43,211 @@ correctly.
 
 * `rewriteZeroIntegerValue(result);`
 
+  Rewrites zero-integer-value with proper tokens
+
+  Before:
+
+  ![rewriteZeroIntegerValue](doc-images/rewriteZeroIntegerValue_input.png) 
+
+  After:
+
+  ![rewriteZeroIntegerValue](doc-images/rewriteZeroIntegerValue_output.png)
+
 * `rewriteIPV4Address(result);`
 
+  Rewrites IPv4address with proper token
+
+  Before:
+
+  ![rewriteIPV4Address](doc-images/rewriteIPV4Address_input.png)
+
+  After:
+
+  ![rewriteIPV4Address](doc-images/rewriteIPV4Address_output.png)
+
 * `orderTokensForLexer(result);`
 
+  Order of tokens sets the priority for lexer when it needs to pick one of the multiple matches.
+  The sooner it occurs in bnf the higher the priority to be matched if multiple tokens can match multiple
+  different characters or strings
+  This method makes sure order looks like this:
+  ![orderTokensForLexer](doc-images/orderTokensForLexer_output.png)
```

Comment:
new line before image otherwise it will be in the same paragraph and braking the formatting